public/app/features/scenes/dashboard/utils/findVizPanel.ts

```diff
@@ -0,0 +1,14 @@
+import { sceneGraph, SceneObject, VizPanel } from '@grafana/scenes';
+
+export function findVizPanel(scene: SceneObject, key: string | undefined): VizPanel | null {
```

Comment:
For future-parity I think we need to rely on panel id rather than scene object key to receive the view/inpect panel.